Source/Scene/Cesium3DTile.js

```diff
@@ -206,14 +231,17 @@ define([
         },
 
         /**
-         * Get the tight oriented bounding box
+         * Get the bounding volume of the tile
          *
-         * @type {Promise}
+         * @type {Object}
          * @readonly
          */
-        orientedBoundingBox : {
+        boundingVolume : {
```

Comment:
Instead of returning `Object`, we should probably introduce an interface, `TileBoundingVolume` (in `Scene` for now).  For an example interface, see [ImageryProvider](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/ImageryProvider.js).